website/docs/r/dbaas_logs_cluster.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "ovh"
+page_title: "OVH: ovh_dbaas_logs_cluster"
+sidebar_current: "docs-ovh-resource-dbaas-logs-cluster"
+description: |-
+  Manage a existing dbaas logs cluster.
+---
+
+# ovh_dbaas_logs_cluster
+
+Reference a DBaaS logs cluster to manipulate ACL.
+
+As LDP cluster can't be created through API, create and delete
+operation are no-op.
+
```

Comment:
Moreover if this feature is only for LDP with enterprise cluster, please add this information :) 

Suggestion:

```
-> This resource can be applied only in `enterprise` cluster type.
```